src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/intrawikisearch/IntraWikiSearchPageObject.java

```diff
@@ -196,18 +196,23 @@ public void selectAllAdvancedOptions() {
     clickAdvancedButton();
     chooseAdvancedOption(0);
     PageObjectLogging.log("selectAllAdvancedOptions", "All advance options are selected", true,
-        driver);
+                          driver);
   }
 
+  /*
+  * Make sure namespace checkboxes are empty, except Articles and Category
+  */
   public void verifyDefaultNamespaces() {
     wait.forElementVisible(advancedField);
     for (int i = 0; i < advancedOptions.size(); i++) {
-      if ("Articles".equals(advancedOptions.get(i).getText())) {
-        Assertion.assertEquals(advancedOptionInputs.get(i).getAttribute("checked"), "true");
-      } else if ("Category".equals(advancedOptions.get(i).getText())) {
-        Assertion.assertEquals(advancedOptionInputs.get(i).getAttribute("checked"), "true");
+
+      String optionName = advancedOptions.get(i).getText();
+      String optionState = advancedOptionInputs.get(i).getAttribute("checked");
+
+      if (optionName.equals("Articles") | optionName.equals("Category")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "Articles" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "Category" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)